.dockerignore

```diff
@@ -0,0 +1,174 @@
+docs
```

Comment:
Add known ignores to project do it once, comprehensively so things aren't overlooked, going forward

Reply:
dockerignore in addition to gitignore